ethcore/src/miner/miner.rs

```diff
@@ -876,6 +877,14 @@ impl Miner {
 			SealingState::NotReady => { self.maybe_enable_sealing(); },
 		}
 	}
+
+	/// Call `update_sealing` if needed
```

Comment:
Maybe document this a bit more thoroughly: what are the conditions used to determine "needed" etc?